lib/ansible/modules/commands/command.py

```diff
@@ -186,9 +186,9 @@ def check_command(module, commandline):
         command = commandline.split()[0]
     command = os.path.basename(command)
 
-    disable_suffix = "If you need to use command because {mod} is insufficient you can add" \
-                     " 'warn: false' to this command task or set 'command_warnings=False' in" \
-                     " ansible.cfg to get rid of this message."
+    disable_suffix = "If you need to use {cmd} because {mod} is insufficient you can add" \
+                     " 'warn: no' to this tasks's args: or set 'command_warnings=False'" \
+                     " in ansible.cfg to get rid of this message."
```

Comment:
```suggestion
                     " in the defaults section of ansible.cfg to get rid of this message."
```

Reply:
What am I supposed to do now? Click the "Commit suggestion" button or go and update/amend my pull request? Or can I just keep my head down and you guys update the pull request to your liking?

Reply:
I would recommend you to amend PR with requested changes.